app/helpers/application_helper/toolbar_chooser.rb

```diff
@@ -667,6 +667,7 @@ def unassigned_configuration_profile_node(nodes)
   end
 
   NO_GTL_VIEW_BUTTONS = %w(chargeback
+                           generic_object
                            generic_object_definition
```

Comment:
I think we can get rid of `generic_object_definition`, there is nowhere that currently uses that layout, as far as I could find, and it probably should've been removed back when I originally wrote it.